.github/workflows/alpha-e2e-scale-trigger.yaml

```diff
@@ -0,0 +1,26 @@
+name: E2EScaleTrigger
+on:
+  schedule:
+    - cron: '7 18 * * *'
+  workflow_run:
+    workflows: [ApprovalComment]
+    types: [completed]
+  workflow_dispatch:
+jobs:
+  resolve:
+    if: (github.repository == 'aws/karpenter' && (github.event_name != 'workflow_run' || github.event.workflow_run.conclusion == 'success')) || github.event_name == 'workflow_dispatch'
+    uses: ./.github/workflows/resolve-args.yaml
+    with:
+      allowed_comment: "scale"
```

Comment:
Does this mean that a `/karpenter scale` comment will trigger both workflows? Is that correct?